lib/l10n/app_en.arb

```diff
@@ -1,5 +1,159 @@
 {
+  "aboutTitle": "About",
+  "aboutDescription": "Detailed information about wallet components",
+  "abstain": "Abstain",
+  "acceleratorAvailableBalance": "Accelerator available balance",
+  "acceleratorDonationsTitle": "Accelerator Donations",
+  "acceleratorStatsTitle": "Accelerator Stats",
+  "acceleratorThanks": "Thank you for supporting the Accelerator",
+  "acceptedStatus": "Accepted",
+  "and": "and",
+  "available": "Available",
   "balance": "Balance",
+  "cancelKey": "Cancel",
```

Comment:
'Cancel' is not a reserved word so, you can use 'cancel' instead of 'cancelKey'. The list of reserved words: https://dart.dev/language/keywords